spec/models/manageiq/providers/ibm_cloud/vpc/network_manager/cloud_subnet_spec.rb

```diff
@@ -5,20 +5,45 @@
     FactoryBot.create(:ems_ibm_cloud_vpc, :provider_region => "us-east")
   end
 
+  let(:cloud_network) do
+    FactoryBot.create(:cloud_network_ibm_cloud_vpc,
+                      :ext_management_system => ems.network_manager)
+  end
+
   let(:cloud_subnet) do
     FactoryBot.create(:cloud_subnet_ibm_cloud_vpc,
                       :ext_management_system => ems.network_manager)
   end
 
-  describe '#raw_delete_cloud_subnet' do
+  describe 'cloud subnet actions' do
     let(:connection) do
       double("ManageIQ::Providers::IbmCloud::CloudTools::Vpc")
     end
 
-    it 'deletes the cloud subnet' do
-      expect(cloud_subnet).to receive(:with_provider_connection).and_yield(connection)
-      expect(connection).to receive(:request).with(:delete_subnet, :id => cloud_subnet.ems_ref)
-      cloud_subnet.raw_delete_cloud_subnet
+    context '#create_cloud_subnet' do
+      it 'creates the cloud subnet' do
+        expect(ems.network_manager).to receive(:with_provider_connection).and_yield(connection)
```

Comment:
Since this line is now expected in both tests you could extract this out to a common before block:
```
before { allow(ems.network_manager).to receive(:with_provider_connection).and_yield(connection) }
```